### DISCUSSION TITLE
We should empasize questions

### DISCUSSION THREAD

**Discussion by lucienmaloney** [acmutd/nebula-guide#4] [Ideas]
I was thinking about ways we could maximize the utility for students. The point of Nebula Guide is to be a place they go to get answers to questions, and our goal is to build a site that makes finding answers to as many questions as possible as easy as possible. So, our situation is this: how do we know what questions students have? I have a feature idea for how we can work on this:

Our site should have a simple form with a single textbox/button pair where users can ask anonymous questions that will get sent to us for review. I think the anonymous part is key here. There are a lot of questions people might want to ask but either are too afraid or nervous to, and a simple easy-to-use form with no personal identification would be unintimidating. We could advertise this feature too and make it a prominent part of our site (maybe put it on the home page?). Of course, we'd have to make it clear that asking a question wouldn't guarantee getting an answer, but I think people would understand that. This feature could work in a positive feedback loop: people will like the draw of anonymous questions which will lead us to a better understanding of students' questions which will make our guide better which will introduce more people to the form. As a bonus, we wouldn't have to do any content moderation or filtering on this form because it would be a thing only we could see.

As for how users get answers to their questions if everything is anonymous, I was thinking that once they submit the form, we direct them to a page that shows off a feed we'll have set up that includes pages sorted by newness and encourage them to check back on it. And maybe we could feature new pages on the home page too somehow (37 new pages created this week/since XXXX-XX-XX!) and also publish new pages regularly in a pdf or something. These are more far-flung ideas, but I think the anonymous form foundation is solid.

**Comment by AvengingPrime** [comment]
Wouldn't this end up just working very similarly to how the UTD subreddit works? The UTD subreddit already works on providing a forum for users to ask questions anonymously for issues that might be more specific than what the guide would have an article about.
